pufferlib/ocean/trash_pickup/README.md

```diff
@@ -0,0 +1,55 @@
+# TrashPickup Environment
```

Comment:
Instead of giving GPT bullets to expand into a page that users will post into GPT to convert back into bullets, just post the bullets

Reply:
Updated (to be pushed).